app/_includes/md/konnect/okta-sso.md

```diff
@@ -0,0 +1,252 @@
+
+    | Field | Value |
+    | ---|--- |
+    | Group claims type | Filter |
+    | Group claims filter | `groups`, select **Matches regex** from the drop-down, then enter `.*` in the field.
+{% endif %}
+
+    This claim tells Okta to reference a subset of Okta groups.
+    In this case, the wildcard (`.*`) value tells Okta to make all groups available for team mapping.
+
+    {:.important}
+    > If the authorization server is pulling in additional groups from
+    third-party applications (for example, Google groups), the `groups` claim
+    cannot find them. An Okta administrator needs to duplicate those groups and
+    re-create them directly in Okta. They can do this by exporting the group in
+    question in CSV format, then importing the CSV file to populate the new group.
+
+1. [Add users to the Okta application](https://help.okta.com/en-us/content/topics/users-groups-profiles/usgp-assign-apps.htm).
+
+{% endnavtab %}
+{% navtab SAML %}
+
+1. Create a [new SAML 2.0 application in Okta](https://help.okta.com/en-us/content/topics/apps/apps_app_integration_wizard_saml.htm?cshid=ext_Apps_App_Integration_Wizard-saml) to manage {{site.konnect_saas}} account integration. Configure the following placeholder settings:
+    * **Single Sign-On URL**: `https://global.api.konghq.com/v2/authenticate/login_path/saml/acs`
+    * **Audience URI (SP Entity ID)**: `https://cloud.konghq.com/sp/SP_ID`
+
+1. Optional: In the **Attribute Statements**, add the following three attributes:
+ 
+    | Name       | Name format  | Value          |
+    |------------|--------------|----------------|
+    | `firstName`  | Unspecified  | user.firstName |
+    | `lastName`   | Unspecified  | user.lastName  |
+    | `email`      | Unspecified  | user.email     |
+
+{% if include.desc == "Dev Portal" %}
+1. Optional: If you want to use group claims for Konnect [developer team mappings](/konnect/dev-portal/access-and-approval/add-teams/), click the **Claims** tab in Okta to [configure a `groups` claim](https://developer.okta.com/docs/guides/customize-tokens-groups-claim/main/#add-a-groups-claim-for-a-custom-authorization-server) and fill in the following fields:
+
+    | Name    | Name format  | Filter          | Filter Value |
+    |---------|--------------|-----------------|--------------|
+    | groups  | Unspecified  | Matches regex   | .*           |
+{% endif %}
+
+1. [Add users to the Okta application](https://help.okta.com/en-us/content/topics/users-groups-profiles/usgp-assign-apps.htm).
+
+1. [Generate a signing certificate](https://help.okta.com/en-us/content/topics/apps/manage-signing-certificates.htm) to use in {{site.konnect_short_name}}.
```

Comment:
There's no place to put the cert in konnect, so I just need to generate it and activate it?